src/App.css

```diff
@@ -37,11 +47,126 @@
   }
 }
 
+
+
 /****Left Hand Menu Styles (other than font styles)****/
-nav div {
+
+#left-menu ul {
+  list-style-type: none;
+}
+
+#left-menu {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  background-color: #D9D9D9;
+}
+
+#left-menu #county-list {
     display: flex;
     flex-direction: column;
-    max-height: 100vh;
+    max-height: calc(100vh - 215px);
+}
+
+#left-menu #data-selector {
+  margin: 1rem auto 1rem;
+  width: calc(100% - 2rem);
+  flex: 1;
+}
+
+#data-selector {
+  background: #FFFFFF 0% 0% no-repeat padding-box;
+  box-shadow: 0px 1px 4px #15223214;
+  border: 1px solid #CED4DA;
+  border-radius: 10px;
+  opacity: 1;
+}
+
+#data-selector h3{
+  margin: 0.5rem auto 0.5rem 1rem;
+  font: normal normal normal 1rem/1.3125rem futura-pt, sans-serif;
+  letter-spacing: 0px;
+  color: #1C752A;
+  opacity: 1;
+}
+
+#data-selector ul {
+  margin: 0px;
+  padding: 0px 0px 0px 1rem;
+}
+
+#data-selector label {
+  font: normal normal normal 0.8125rem/1.0625rem futura-pt;
+  letter-spacing: 0px;
+  color: #1C752A;
+  opacity: 1;
+  font-weight: 400;
+}
```

Comment:
![Screen Shot 2021-05-19 at 10 31 12 PM](https://user-images.githubusercontent.com/35851648/118910088-f3b0a000-b8f1-11eb-9ba3-175d39397412.png)

Maybe it's related to the above component not being up to design specs yet, but can your component be a little bit wider and with less margin at the bottom (under the last list item)? Also maybe we can give the whole left column a gray background like this so there is not white under it?

Reply:
Also bolder and more centered for the title? [https://xd.adobe.com/view/915910be-fb03-46d4-a5df-e8e1109c8e31-c40a/screen/5aa15b4b-e156-47f5-938a-a9168feb8866/](url)

Reply:
Hey, so I've made a quite a lot of directory structure changes (the thing Ritesh and I spoke about during the meetings) and also fixed the CSS issues you pointed out. Let me know what you think.

Small points: 
1. The bottom left container looks better now in terms of size, but it is dependent on a hardcoded height of the "ZoomToBoundsMenu" (see App.css, line 37). I don't have a better solution atm :/

2. The width of the selector is too big, but it is based on the width of the whole column, which is not in line with the size of the prototype. I think that might be a different PR so I want this merged in first so it doesn't keep growing in all directions.

Hope this works better! Let men know if there are more things to tweak.